### ISSUE TITLE
Not able to reproduce the results for Conceptual Captions 3M

### ISSUE THREAD

**Issue by raja-7-c** [mlfoundations/open_clip#1025]
I am running the following code by utilising the cc3M dataset from "pixparse/cc3m-wds"

python -m open_clip_train.main \
         --train-data="cc3m-wds/cc3m-train-{0000..0575}.tar" \
         --train-num-samples 2905954 \
         --val-data="cc3m-wds/cc3m-validation-{0000..0015}.tar" \
         --val-num-samples 13443 \
         --dataset-type webdataset \
         --batch-size 256 \
         --warmup 2000 \
         --epochs 10 \
         --lr 5e-4 \
         --precision amp \
         --workers 6 \
         --model "roberta-ViT-B-32" \
         --lock-text \
         --lock-text-unlocked-layers 10 \
         --name "10_unfrozen" \
         --report-to "wandb" \
         --wandb-project-name "open-clip-brdcst" \
         --name "openclip-cc3m"

I am attaching the images for training loss, what could be the reasons for the following plots.....

![Image](https://github.com/user-attachments/assets/23adb441-9019-43b6-a885-8888b0bfdc50)

for the reference, i have also attached the expected loss curve

![Image](https://github.com/user-attachments/assets/2b112c5d-1538-4af9-8015-3f189c234c4b)

**Comment by rwightman** [comment]
@raja-7-c I don't believe there was an expected loss curve posted for the roberta LM variant, just the normal ones. Using a pretrained language encoder can actually get a bit tricky to find the right balance, I feel the command line for that in the README was just an example of using such an encoder, not necessarily an optimal setting. The single GPU w/ 256 batch size for instance, isn't going to be amazing.

**Comment by raja-7-c** [comment]
Thanks a lot for open-sourcing the code and for your quick reply! Yeah, under normal conditions, I was getting quite the same plot as expected with effectively higher batch sizes.

![Image](https://github.com/user-attachments/assets/2050209e-31ae-46ab-9d00-6ea2922109d1)
